### PR TITLE
Feature/nijo 955 data provider to suppliers5 - Step 5

### PR DESCRIPTION
**Pull request by mingazhev**
Renamed method params and method vars + removed passing supplier to etgwebhook service

### CODE REVIEW

**Review by hasmikharutyunyan — APPROVED** [review]

File: Api/Services/Accommodations/Availability/Steps/BookingEvaluation/BookingEvaluationService.cs [hunk 1]
```diff
@@ -94,14 +94,14 @@ Task SaveToCache(DataWithMarkup<RoomContractSetAvailability?> responseWithDeadli
...
                 var finalRoomContractSetId = responseWithDeadline.Data.Value.RoomContractSet.Id;
                 return _bookingEvaluationStorage.Set(searchId, resultId, finalRoomContractSetId, DataWithMarkup.Create(responseWithDeadline.Data.Value, 
-                    responseWithDeadline.Policies), result.DataProvider);
+                    responseWithDeadline.Policies), result.Supplier);
             }
 
 
             RoomContractSetAvailabilityInfo? ToDetails(DataWithMarkup<RoomContractSetAvailability?> availabilityDetails)
             {
                 var provider = settings.IsDataProviderVisible
```
Comment (hasmikharutyunyan) [inline Api/Services/Accommodations/Availability/Steps/BookingEvaluation/BookingEvaluationService.cs:103]: isSupplierVisible?

File: Api/Services/Accommodations/Availability/Steps/WideAvailabilitySearch/WideAvailabilitySearchTask.cs [hunk 2]
```diff
@@ -78,8 +78,8 @@ await GetAvailability(request, languageCode)
             async Task<Result<EdoContracts.Accommodations.Availability, ProblemDetails>> GetAvailability(AvailabilityRequest request,
                 string languageCode)
             {
-                var saveToStorageTask = _storage.SaveState(searchId, ProviderAvailabilitySearchState.Pending(searchId), provider);
-                var getAvailabilityTask = dataProvider.GetAvailability(request, languageCode);
+                var saveToStorageTask = _storage.SaveState(searchId, SupplierAvailabilitySearchState.Pending(searchId), provider);
```
Comment (kirillta) [inline Api/Services/Accommodations/Availability/Steps/WideAvailabilitySearch/WideAvailabilitySearchTask.cs:81]: provider

**Review by kirillta — APPROVED** [review]

File: Api/Services/Accommodations/Availability/Steps/BookingEvaluation/BookingEvaluationService.cs (hunk 1 above)
Comment (mingazhev) [reply Api/Services/Accommodations/Availability/Steps/BookingEvaluation/BookingEvaluationService.cs:103]: This is saved to database as JSON, so will be fixed later

File: Api/Services/Accommodations/Availability/Steps/WideAvailabilitySearch/WideAvailabilitySearchState.cs [hunk 3]
```diff
@@ -53,7 +53,7 @@ public static WideAvailabilitySearchState FromProviderStates(Guid searchId, IEnu
             return new WideAvailabilitySearchState(searchId, overallState, totalResultsCount, errors);
 
 
-            static AvailabilitySearchTaskState CalculateOverallState(Dictionary<Suppliers, ProviderAvailabilitySearchState> providerSearchStates)
+            static AvailabilitySearchTaskState CalculateOverallState(Dictionary<Suppliers, SupplierAvailabilitySearchState> providerSearchStates)
```
[the remark this answers is not in the document]
Comment (mingazhev) [reply Api/Services/Accommodations/Availability/Steps/WideAvailabilitySearch/WideAvailabilitySearchState.cs:56]: In the next PR

File: Api/Services/Accommodations/Availability/Steps/WideAvailabilitySearch/WideAvailabilitySearchTask.cs (hunk 2 above)
Comment (mingazhev) [reply Api/Services/Accommodations/Availability/Steps/WideAvailabilitySearch/WideAvailabilitySearchTask.cs:81]: In the next PR

**Pull request merged**